Source/Example Measurements/CancellableMeasurement/README.md

```diff
@@ -0,0 +1,21 @@
+# CancellableMeasurement
+
+This is a MeasurementLink example that demonstrates a measurement
+that takes a specified length of time and can be cancelled while it is running.
+
+It will run for the amount of time specified with the `Delay (sec)` in the UI unless the `STOP` button is clicked.
+
+Once the measurement completes or is cancelled, it will return a random number.
```

Comment:
The return values are ignored when the measurement is canceled.

Reply:
Updated to take out the 'cancelled'.